dev-cpp/yaml-cpp/files/yaml-cpp-0.7.0-install-paths.patch

```diff
@@ -0,0 +1,34 @@
+From 4f5b5ba19ece906252cfc90b7cd51035e21c7118 Mon Sep 17 00:00:00 2001
```

Comment:
Please follow https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/#clean-patch-howto

`scrub-patch` command from app-portage/iwdevtools can easily achieve this

Reply:
not resolved yet, this header is not useful